controllers/chat/ChatController.js

```diff
@@ -94,8 +95,7 @@ module.exports = {
       let {last_fetch_date = null} = req.query;
       let filter = {members: {$in: [req.userId]}};
       if (last_fetch_date) {
-        last_fetch_date = new Date(last_fetch_date);
-        last_fetch_date = last_fetch_date.toISOString();
+        last_fetch_date = moment.utc(last_fetch_date).toDate().toISOString();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conversion of `last_fetch_date` to UTC using `moment` is a good approach to ensure consistency across different time zones. However, the use of `toDate()` before `toISOString()` seems unnecessary since `moment.utc().toISOString()` would directly give you the ISO string in UTC.
```diff
- last_fetch_date = moment.utc(last_fetch_date).toDate().toISOString();
+ last_fetch_date = moment.utc(last_fetch_date).toISOString();
```
If you decide to switch to `date-fns`, you can use `parseISO` to parse the date string and `formatISO` to convert it back to an ISO string in UTC.
```diff
- last_fetch_date = moment.utc(last_fetch_date).toDate().toISOString();
+ last_fetch_date = formatISO(parseISO(last_fetch_date));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->